### ISSUE TITLE
CI build broken on 4.13.0

### ISSUE THREAD

**Issue by ursetto** [ursetto/chickadee#3]
Building in an ubuntu-latest runner on 4.13.0 fails test in spiffy-request-vars:

```
(get "/wrv-as-symbol?var=") .......................................... [ FAIL]
```

I can't reproduce this on a 4.13.0 build on macOS M1, though.

**Comment by ursetto** [comment]
Works fine when run repeatedly on macOS x86_64. Unable to trigger after 100+ runs (rebuilding only spiffy-request-vars, not everything though).

Worked fine with `act` using the 22.04 runner image: `act -j test -s GITHUB_TOKEN=$GITHUB_TOKEN -P ubuntu-latest=ghcr.io/catthehacker/ubuntu:runner-22.04`. This means the chicken-versions build of 4.13.0 looks OK.

However, failed with `act` when run a second time! It failed twice in maybe 10 tries.

Additionally, worked on ubuntu-latest on GitHub on a second try!

On runner-22.04 manual running with chicken-versions build, running `chicken-install -r spiffy-request-vars -test` and then running `chicken-install -test` in a loop (downloading nothing and rebuilding spiffy-request-vars only) fails on average every few runs. Sometimes on try 6, sometimes try 1 or 2.

Fails similarly with manually built 4.13.0 on fresh Ubuntu 22.04 VM on ARM64 (M1).

We can trigger this without even building; simply run `csi -s run.scm` in a loop in the tests directory after the egg is installed.

A heisenbug, but unknown if it is limited to the chicken-versions build, or not.

**Comment by ursetto** [comment]
This is apparently a known issue with 4.x: http://bugs.call-cc.org/ticket/991#comment:4

It's unknown if this was ever fixed, and also unknown if it propagated to 5.x.

I can postulate a possible method of returning ":" from an empty symbol:

```
> (print (string->keyword (symbol->string '||)))
:
```

**Comment by ursetto** [comment]
Fails intermittently on:

- Ubuntu 22.04 x86_64 on GH runner, local act (Docker) runner
- Ubuntu 22.04 M1 on UTM VM

Succeeds on:

- macOS x86_64 and M1, bare metal

**Comment by ursetto** [comment]
Was fixed by #4.
